benchmarks/README.md

```diff
@@ -0,0 +1,133 @@
+# AI on GKE Benchmarking
+
+This framework enables you to run automated benchmarks on GKE for AI workloads via Terraform automation.
+You can find the current set of supported cluster deployments under `infra/` and the supported inference servers
+under `inference-server/`.
+
+Note that you can run any stage separate from another. For example, you can deploy your own inference server
+on a cluster created via these terraform scripts. And vice versa, you can deploy the inference servers
+via these terraform scripts on a Standard cluster that you've created yourself.
+
+## Pre-requisites
+
+### Terraform
+
+Install Terraform by following the documentation at https://developer.hashicorp.com/terraform/tutorials/aws-get-started/install-cli.
+This requires a minimum Terraform version of 1.7.0
+
+## Deploy and serve an ML model
+
+This section goes over an end to end example to deploy and serve the Falcon 7b model using (TGI)[https://huggingface.co/docs/text-generation-inference/en/index] on a Standard GKE Cluster with GPUs.
+
+Each step below has more details in their respective directoy README.md. It is recommended
+that you read through the available options at least once when testing your own models.
+
+### Create the cluster
+
+Set up the infrastructure by creating a GKE cluster with appropriate accelerator
+configuration.
+
+To create a GPU cluster, run
+```
+# Stage 1 creates the cluster.
+cd infra/gpu/stage-1
```

Comment:
@achandrasekar something worth updating with removing the gpu/ folder

We need to update top level readme to remove the infra/gpu/... references

Reply:
Done